samples/test/datasets.test.js

```diff
@@ -105,4 +111,34 @@ describe('Datasets', () => {
     const [exists] = await bigquery.dataset(datasetId).exists();
     assert.strictEqual(exists, false);
   });
+
+  // Only delete a resource if it is older than 24 hours. That will prevent
+  // collisions with parallel CI test runs.
+  function isResourceStale(creationTime) {
+    const oneDayMs = 86400000;
```

Comment:
Non-blocking feedback, I like to spell this out as `24 * 60 * 60 * 1000` to make it easier to reason about